README.md

```diff
@@ -8,8 +8,20 @@
 
 ### Demo
 
+Installation & Help message
+[![asciicast](https://asciinema.org/a/616538.svg)](https://asciinema.org/a/616538)
+
+Plain JSON files diff:
 [![asciicast](https://asciinema.org/a/602583.svg)](https://asciinema.org/a/602583)
+
+Plain YAML files diff:
 [![asciicast](https://asciinema.org/a/604893.svg)](https://asciinema.org/a/604893)
+
+Nested JSON files diff:
 [![asciicast](https://asciinema.org/a/612701.svg)](https://asciinema.org/a/612701)
```

Comment:
this asciinema has runs for nested and plain JSON files, consider removing plain json diff from this asciinema (so you have to rerecord it)